### PR TITLE
Improve pinch event

### PR DESCRIPTION
**Pull request by bergwerf**
# Description

Address #1190.

@dsehnal I took a stab at it. What do you think, and do you have suggestions for improvements? My other PR #1191 adds a test page that creates a grid of four Canvas3Ds which you could use to test this out. If you think this is a good solution direction, I will check off the points below.

## Actions

- [x] Added description of changes to the `[Unreleased]` section of `CHANGELOG.md`
- [x] Updated headers of modified files
- [x] Added my name to `package.json`'s `contributors`
- [ ] (Optional but encouraged) Improved documentation in `docs`

### CODE REVIEW

**Review by dsehnal — COMMENTED** [review]
A few cosmetic comments, but otherwise I think this is fine.

Can you please include the info in the headers and changelog too?

File: src/mol-canvas3d/controls/trackball.ts [hunk 1]
```diff
@@ -614,18 +613,30 @@ namespace TrackballControls {
...
+                    if (pan) {
+                        Vec2.copy(_panStart, mouseOnScreenVec2);
+                        Vec2.copy(_panEnd, _panStart);
+                    }
+                } else {
+                    if (pan) {
+                        Vec2.copy(_panEnd, mouseOnScreenVec2);
+                    }
+                    if (zoom) {
+                        _zoomEnd[1] += p.gestureScaleFactor * -fractionDelta;
```
[the remark this answers is not in the document]
Comment (bergwerf) [reply src/mol-canvas3d/controls/trackball.ts:634]: I think I computed the fractionDelta the wrong way around.

**Comment by bergwerf** [comment]
@dsehnal I integrated your feedback.

**Review by dsehnal — APPROVED** [review]
LGTM, thanks.

**Pull request merged**